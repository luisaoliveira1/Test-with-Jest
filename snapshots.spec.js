// Snapshot is a representation of data in the string format
// O snapshot deverá ir no repositório do git junto com o código.
// para atualizar um snapshot, é necessário passar um yarn test nomeDoSpec -u
const user = {
  name: 'Luisa Oliveia',
  age: 24,
  job:'Frontend Developer'
}

test( 'user matches', () => {

  expect(user).toMatchSnapshot()
})