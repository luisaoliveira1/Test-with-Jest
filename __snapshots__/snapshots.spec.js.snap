// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`user matches 1`] = `
Object {
  "age": 24,
  "job": "Frontend Developer",
  "name": "Luisa Oliveia",
}
`;
